domains/ajemina.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "My portfolio",
+  "repo": "https://github.com/raphaeljay/resume.me",
+  "owner": {
+    "username": "raphaeljay"
+    "email": "ralphajemina4live@gmail.com"
+  },
+  "records": {
+    "CNAME": "ajemina.github.io"
```

you dont own the github page
```suggestion
    "CNAME": "raphaeljay.github.io"
```